assets/js/components/ClusterDetails/ClusterInfoBox.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { ClusterInfoBox } from './ClusterInfoBox';
+
+describe('Cluster Info Box', () => {
+  [
+    {
+      haScenario: 'hana_scale_up',
+      haScenarioText: 'HANA scale-up',
+      provider: 'aws',
+      providerText: 'AWS',
+    },
+    {
+      haScenario: 'hana_scale_out',
+      haScenarioText: 'HANA scale-out',
+      provider: 'aws',
+      providerText: 'AWS',
+    },
+    {
+      haScenario: 'hana_scale_up',
+      haScenarioText: 'HANA scale-up',
+      provider: 'azure',
+      providerText: 'Azure',
+    },
+    {
+      haScenario: '',
+      haScenarioText: 'Unknown',
+      provider: 'gcp',
+      providerText: 'GCP',
+    },
+    {
+      haScenario: 'unknown',
+      haScenarioText: 'Unknown',
+      provider: 'kvm',
+      providerText: 'KVM',
+    },
+    {
+      haScenario: 'hana_scale_up',
+      haScenarioText: 'HANA scale-up',
+      provider: 'nutanix',
+      providerText: 'Nutanix',
+    },
+  ].forEach(({ haScenario, haScenarioText, provider, providerText }) => {
+    it(`should display ${providerText} as the provider and HA Scenario: ${haScenarioText}`, () => {
+      render(<ClusterInfoBox haScenario={haScenario} provider={provider} />);
```

Comment:
This is still a bit risky. You can have the scenario where all the components are rendered, and you are checking in other screen part:
```
{ component } = render(<ClusterInfo...
expect(component.getAll...
```